android-components/components/feature/search/src/test/java/mozilla/components/feature/search/telemetry/incontent/InContentTelemetryTest.kt

```diff
@@ -240,7 +240,7 @@ class InContentTelemetryTest {
 
     @Test
     fun `GIVEN a Baidu sap-follow-on WHEN trackPartnerUrlTypeMetric is called THEN emit an appropriate IN_CONTENT_SEARCH fact`() {
```

Comment:
We're not recording tagged codes for Baidu currently, so this sub-test can be removed.